modules/living_spaces_simple_permissions/living_spaces_simple_permissions.module

```diff
@@ -132,3 +135,22 @@ function living_spaces_simple_permissions_get_permissions_by_module($type, $modu
 
   return $permissions;
 }
+
+/**
+ * Implements hook_living_spaces_breadcrumbs_info().
+ */
+function living_spaces_simple_permissions_living_spaces_breadcrumbs_info(RouteMatchInterface $route_match, Breadcrumb $breadcrumb = NULL) {
+  $applies = FALSE;
+
+  $query = \Drupal::request()->query->all();
+  if ('user.admin_permissions' == $route_match->getRouteName() && isset($query['filter_roles']) && $query['filter_roles']) {
```

Comment:
we can use here
`if ('user.admin_permissions' == $route_match->getRouteName() && !empty($query['filter_roles'])) {`
and think it's beautiful

Reply:
done